Web Crawler/src/webcrawler/MyCrawler.java

```diff
@@ -0,0 +1,41 @@
+package webcrawler;
+
+import java.util.Set;
+
+public class MyCrawler {
+	private static final int MAX_PAGES_TO_SEARCH = 10;
```

Comment:
That's a constant indicating how many pages we want to search. We surely could increase this value. However, the more we dig in, the chance of finding email addresses will get low. Cause I already put the webpages highly valuable into the proirQueue. What's more, that would take too long for you to review, right? @clee-jana